libs/database/src/lib/services/ads.service.ts

```diff
@@ -155,4 +156,22 @@ export class AdsService {
       updatedAt: campaign.updatedAt,
     } as AdsCampaignResponseDto;
   };
+
+  async listAds(
+    { _id }: Account,
+    { sinceId, untilId, maxResults }: PaginationQuery
+  ) {
+    let filter: FilterQuery<AdsCampaign> = {
+      owner: _id,
+    };
+
+    filter = await createCastcleFilter(filter, {
+      sinceId,
+      untilId,
+    });
```

Comment:
```suggestion
    const filter: FilterQuery<AdsCampaign> = createCastcleFilter({ owner: _id }, {
      sinceId,
      untilId,
    });
    
    // or
    const filter = createFilterQuery<AdsCampaign>(sinceId, untilId);
    
    filter.owner = _id;
```
avoid reassigning params (let)